sdks/java/io/hcatalog/src/test/java/org/apache/beam/sdk/io/hcatalog/HiveDatabaseTestHelper.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.beam.sdk.io.hcatalog;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import org.apache.beam.sdk.io.common.DatabaseTestHelper;
+
+/**
+ * Helper for creating connection and test tables on hive database via JDBC driver.
+ */
+
+public class HiveDatabaseTestHelper {
+  private static Connection con;
+  private static Statement stmt;
+
+  HiveDatabaseTestHelper(String hiveHost,
+                         Integer hivePort,
+                         String hiveDatabase) throws SQLException {
```

Comment:
Maybe throw general `Exception` because anyway it is propagated by its sole user..

Reply:
OK, good point.